lib/browser/guest-view-manager.ts

```diff
@@ -192,7 +208,8 @@ const attachGuest = function (event, embedderFrameId, elementInstanceId, guestIn
       ? parseWebViewWebPreferences(params.webpreferences)
       : null;
 
-  const webPreferences = {
+  // Typing satisfy's no index signature error on webPreferences in for loop
+  const webPreferences: { [key: string]: any } = {
```

Comment:
```suggestion
  const webPreferences: Electron.WebPreferences = {
```

Reply:
Changing to this typing causes the error that I mention in my comment on line 211.